docs/plugins.md

```diff
@@ -144,3 +146,12 @@ def platform
  )
 end
 ```
+Example config for transport that supports `unix` and `windows` OS,
+
+```Ruby
+def platform
+  Train::Platforms.name("local-rot13").in_family("unix")
```

Comment:
I did not understood what "local-rot13" here is. Also what is `Example config`.

Reply:
This is the recommended sample train plugin https://github.com/inspec/train/blob/main/examples/plugins/train-local-rot13/lib/train-local-rot13/platform.rb

Reply:
`Example config` == `Configuration Example`